Core/include/Acts/Seeding/HoughTransformUtils.hpp

```diff
@@ -109,23 +109,33 @@ class HoughCell {
   YieldType nLayers() const { return m_nLayers; }
   /// @brief access the number of unique hits compatible with this cell
   YieldType nHits() const { return m_nHits; }
-  /// @brief access the set of layers compatible with this cell
-  const std::unordered_set<unsigned>& layers() const { return m_layers; }
-  /// @brief access the set of unique hits compatible with this cell
-  const std::unordered_set<identifier_t>& hits() const { return m_hits; }
+  /// @brief access the span of layers compatible with this cell
+  std::span<const unsigned, std::dynamic_extent> getLayers() const;
+  // /// @brief access the pan of unique hits compatible with this cell
+  std::span<const identifier_t, std::dynamic_extent> getHits() const;
+
   /// @brief reset this cell, removing any existing content.
   void reset();
 
  private:
   /// data members
 
-  YieldType m_nLayers =
-      0;                  // (weighted) number of layers with hits on this cell
-  YieldType m_nHits = 0;  // (weighted) number of unique hits on this cell
-  std::unordered_set<unsigned> m_layers =
-      {};  // set of layers with hits on this cell
-  std::unordered_set<identifier_t> m_hits =
-      {};  // set of unique hits on this cell
+  YieldType m_nLayers{0};  // (weighted) number of layers with hits on this cell
+  YieldType m_nHits{0};    // (weighted) number of unique hits on this cell
+
+  // index for the hits -- keeps track of vector's size
+  std::size_t m_ihit{0};
+  // index for the layers -- keeps track of vector's size
+  std::size_t m_ilayer{0};
```

Comment:
- slightly inconsistent with above `nLayer` `nHits` -> `iHit`
- for the doc to work it should be `///` I believe